### ISSUE TITLE
Option to make the AL Object Browser the default 'editor' for .app files

### ISSUE THREAD

**Issue by rvanbekkum** [anzwdev/al-code-outline#145]
Hi @anzwdev,
Not sure if this is (already) possible or not with the [custom editor API](https://github.com/Microsoft/vscode-extension-samples/tree/master/custom-editor-sample), but do you think it would be possible to have an option to set the AL Object Browser as default 'editor' for .app files?

**Comment by anzwdev** [comment]
Hi @rvanbekkum 
Yes, it will be posible and it is already on my list of things to do, I just wanted to implement text based custom editor first as that API has been released a bit earlier than binary editors.

**Comment by anzwdev** [comment]
I had to update references to the visual studio libraries used by my extension to be able to use custom binary editors. It requires VS Code 1.47 or newer now. Extension adds a default read only binary editor for *.app files, it is opened when you click on the file. "Open in AL Object Browser" context menu item is still available to give people time to switch to the new way of viewing these files. New functionality is available in the extension update that I've just released.
